lib/lambda/submit/indexMissingAttachments.test.ts

```diff
@@ -0,0 +1,193 @@
+import { submit } from "./index";
+import { APIGatewayEvent } from "node_modules/shared-types";
+import { describe, it, expect, vi, afterEach, beforeEach } from "vitest";
+import { bma } from "mocks/data/submit/baseMissingAttachments";
+import { getRequestContext } from "mocks";
+
+vi.mock("kafkajs", () => {
+  const producer = {
+    connect: vi.fn(),
+    send: vi.fn(),
+    disconnect: vi.fn(),
+  };
+  const kafka = {
+    producer: () => producer,
+  };
+  return {
+    Kafka: vi.fn(() => kafka),
+    Producer: vi.fn(() => producer),
+  };
+});
+describe("submit Lambda function missing objects in event", () => {
+  let brokerString: string | undefined;
+  beforeEach(() => {
+    brokerString = process.env.brokerString;
+    process.env.brokerString = "broker1,broker2";
+  });
+
+  afterEach(() => {
+    process.env.brokerString = brokerString;
+  });
+
+  it("should start to create an capitated ammendment event", async () => {
```

Comment:
can we have a little more info in these test names? like "should fail creating a capitated amendment event because..."